### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by Ethan-Bit-256** [mapeditor/tiled#3079] [Q&A]
How do I help translate? I want to add a translation in the software for my native language Swahili

**Comment by bjorn** [comment]
It's great that you'd like to help add another translation to Tiled!

Please find general instructions at https://github.com/mapeditor/tiled/wiki/Translating-Tiled. The easiest way, which does not require installation of any software, is to rely solely on Weblate, but there are also some instructions about doing it locally with Qt Linguist.

Let me know if you have any questions. :-)
